docs/internals/organization.txt

```diff
@@ -168,8 +168,7 @@ Role
 The steering council is a group of experienced contributors who:
 
 - provide oversight of Django's development and release process,
-- assist in setting the direction of feature development and releases,
-- take part in filling certain roles, and
```

Comment:
I have removed "take part in filling certain roles" as I think it is too vague to add value to a role description
I can change this to something specific if required

Reply:
Although I don't think this is clear, this is quoted directly from [DEP-10](https://github.com/django/deps/blob/main/final/0010-new-governance.rst#repurposed-role-technical-board). So I will leave this for now as this should be aligned

Reply:
https://github.com/django/deps/pull/93

Reply:
Updated the DEP and now aligned this here